wai-saml2.cabal

```diff
@@ -20,6 +20,7 @@ build-type:     Simple
 extra-source-files:
     README.md
     CHANGELOG.md
+    tests/data/google.xml.base64
```

Comment:
This file doesn't seem to be included in the PR - did you mean to include it or exclude this line?

Reply:
Hmm, it looks like `hpack` added this line, which isn't intended